template/registry/number.go

```diff
@@ -189,9 +189,7 @@ func parseNumberOptions(opts Options) (*numberOptions, error) {
 	}
 
 	// The maximum number of significant digits to use.
-	const maxSignificantDigits = 21
-
-	options.MaximumSignificantDigits, err = opts.GetPositiveInt("maximumSignificantDigits", maxSignificantDigits, nil)
+	options.MaximumSignificantDigits, err = opts.GetPositiveInt("maximumSignificantDigits", -1, nil)
```

Comment:
Shouldn't the fallback in "get **positive** integer" function be, like, positive? :)

Reply:
Fixed, renamed to `GetInt` and updated the func signature.